src/common/multihash.c

```diff
@@ -86,14 +87,17 @@ daos_hashtype2contprop(enum DAOS_HASH_TYPE daos_hash_type)
  * and other accelerator support.
  */
 static struct hash_ft **algo_table = isal_hash_algo_table;
-
+static struct hash_ft **qat_algo_table = qat_hash_algo_table;
 struct hash_ft *
 daos_mhash_type2algo(enum DAOS_HASH_TYPE type)
 {
 	struct hash_ft *result = NULL;
 
 	if (type > HASH_TYPE_UNKNOWN && type < HASH_TYPE_END) {
-		result = algo_table[type - 1];
+		if (qat_algo_table[type-1] != NULL)
```

Comment:
So with qat enabled, it will always be chosen over isal?

Reply:
For now, QAT has high priority than isal, but in future we may also think of some kind of options to allow user specify if QAT is preferred.